frontend/src/utils/ValidateURL.tsx

```diff
@@ -0,0 +1,9 @@
+import { useInstallationContext } from 'components/Contexts/InstallationContext'
+import { config } from 'config'
+
+export function validateURL() {
+    const { installationCode, installationName } = useInstallationContext()
```

Comment:
Does using contexts inside non-component functions not cause an error/warning in the frontend?

Reply:
Could we rename the function so that it's clear that it will navigate away, and not just validate the inspection?